### PR TITLE
Update cascading front matter to use sequence form

### PR DESCRIPTION
**Pull request by jdbaldry**
The map form does not override the sequence form that is used in the website repository to specify the default labels.
For more information, refer to https://github.com/grafana/writers-toolkit/pull/234.

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/administration/_index.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-    products:
-      - cloud
-      - enterprise
-      - oss
+  - _target:
+      path: /docs/grafana/*/administration/**
+    labels:
+      products:
+        - cloud
+        - enterprise
```
Comment (chri2547) [inline docs/sources/administration/_index.md:8]: Does this mean that all docs under Admin receive the Enterprise label? This should not be the case.

**Review by chri2547 — COMMENTED** [review]
Identified a few more files that we should add to this PR.

**Comment by jdbaldry** [comment]
Thanks Chris, I have updated the PR to correct the labels on the pages you identified and have centralized all the cascading front matter into the projects root index file so that all product definitions can be found in one place.

**Review by chri2547 — APPROVED** [review]
Awesome, thank you!

File: docs/sources/_index.md [hunk 2]
```diff
@@ -3,6 +3,192 @@ aliases:
...
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-request-security/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
```
Comment (imatwawana) [inline docs/sources/_index.md:166]: @chri2547 @jdbaldry - there are notes in the docs that seem to indicate the labels should be different for these two pages.

File: docs/sources/_index.md [hunk 3]
```diff
@@ -3,6 +3,192 @@ aliases:
...
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/export-logs/** }
+    labels:
+      products:
+        - cloud
+        - oss
```
Comment (imatwawana) [inline docs/sources/_index.md:178]: @chri2547 @jdbaldry - there are notes in the docs that seem to indicate the labels should be different for this page.

File: docs/sources/_index.md [hunk 4]
```diff
@@ -3,6 +3,192 @@ aliases:
...
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/whatsnew/** }
+    labels:
+      products:
+        - cloud
+        - oss
```
Comment (imatwawana) [inline docs/sources/_index.md:191]: @chri2547 @jdbaldry - I'm not clear on why this wouldn't apply to Enterprise. Maybe I'm missing something?

**Review by imatwawana — APPROVED** [review]
I left a questions for 4 of the pages, but otherwise this LGTM!

File: docs/sources/_index.md (hunk 4 above)
Comment (imatwawana) [reply docs/sources/_index.md:196]: Per Mitch, we should use all three labels here.

```suggestion
  - _target: { path: /docs/grafana/*/whatsnew/** }
    labels:
      products:
        - cloud
        - enterprise
        - oss
```

**Pull request merged**